platform/lang-impl/src/com/intellij/build/BuildTreeConsoleView.java

```diff
@@ -232,12 +244,17 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
   private ExecutionNode getOrMaybeCreateParentNode(@NotNull BuildEvent event) {
     ExecutionNode parentNode = event.getParentId() == null ? null : nodesMap.get(event.getParentId());
     if (event instanceof MessageEvent) {
-      parentNode = createMessageParentNodes((MessageEvent)event, parentNode);
+      ExecutionNode node = parentNode;
```

Comment:
Why do we need this variable here?

Reply:
It used not to be ExecutionNode originally. I'll remove it.

Reply:
thanks you

Reply:
removed